src/components/views/settings/SecureBackupPanel.js

```diff
@@ -367,14 +367,14 @@ export default class SecureBackupPanel extends React.PureComponent {
             </>;
 
             actions.push(
-                <AccessibleButton kind="primary" onClick={this._restoreBackup}>
+                <AccessibleButton kind="primary" onClick={this._restoreBackup} key="restoreButton">
```

Comment:
Hmm, I have made a similar but different fix in my own PR https://github.com/matrix-org/matrix-react-sdk/pull/5219... 🙃 Maybe let's remove this change from here, since it seems not really related to this PR?